superagent-ai/superagent
# UI: Showing Ingestion status

## Summary
Also fixes #892 

Depends on [#91](https://github.com/superagent-ai/super-rag/pull/91)


## Steps to test
- Make sure you have set `NEXT_PUBLIC_SUPERRAG_API_URL` in .env e.g. `NEXT_PUBLIC_SUPERRAG_API_URL=http://localhost:1234/api/v1`
- Make sure your superrag branch is `feat/add-message-queue`

https://github.com/homanp/superagent/assets/67149699/29ff56d5-f4ed-46ff-90de-0b3dcce321fc
